dlc-node-test/src/test/scala/org/bitcoins/dlc/node/DLCServerTorTest.scala

```
@@ -20,27 +21,21 @@ import java.net.InetSocketAddress
 import scala.concurrent.duration.DurationInt
 import scala.concurrent.{Future, Promise}
 
-class DLCServerTorTest extends BitcoinSActorFixtureWithDLCWallet {
-
-  val torProxyAddress = new InetSocketAddress("localhost", 9050)
-  val torControlAddress = new InetSocketAddress("localhost", 9051)
-  val torProxyEnabled: Boolean = portIsBound(torProxyAddress)
-  val torControlEnabled: Boolean = portIsBound(torControlAddress)
+class DLCServerTorTest
+    extends BitcoinSActorFixtureWithDLCWallet
+    with CachedTor {
 
   override type FixtureParam = FundedDLCWallet
 
+  implicit val conf: BitcoinSAppConfig = getFreshConfig
+
   override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
-    // Skip on CI as we don't have access to tor control port
-    if (EnvUtil.isCI) {
-      FutureOutcome.succeeded
-    } else withFundedDLCWallet(test, getBIP39PasswordOpt())(getFreshConfig)
+    if (TorUtil.torEnabled) {
+      withFundedDLCWallet(test, getBIP39PasswordOpt())
+    } else FutureOutcome.succeeded
```

Hmm, why succeed? Isn't it implied that this test case _needs_ tor to be able to run? It seems like we should fail loudly if tor is not enabled? Otherwise you have trivial silent failures?

---

We just want to skip this test if tor isn't enabled. I think this is the one way to do it, maybe can add a log